res/values/strings.xml

```diff
@@ -530,6 +530,18 @@
     <string name="systemmsg_action_by_me">%1$s by me.</string>
     <!-- Translators: %1$s will be replaced by sth. as "member xy added" (trailing full-stop removed). %2$s will be replaced by the name/addr of the person who did this action. -->
     <string name="systemmsg_action_by_user">%1$s by %2$s.</string>
+    <string name="systemmsg_unknown_sender_for_chat">Unknown Sender for this chat. See \'info\' for more details.</string>
+    <string name="systemmsg_subject_for_new_contact">Message from %1$s</string>
+    <string name="systemmsg_failed_sending_to">Failed to send message to %1$s.</string>
+
+    <string name="systemmsg_ephemeral_timer_disabled">Message deletion timer is disabled.</string>
+    <string name="systemmsg_ephemeral_timer_enabled">Message deletion timer is set to %1$s s.</string>
+    <string name="systemmsg_ephemeral_timer_minute">Message deletion timer is set to 1 minute.</string>
+    <string name="systemmsg_ephemeral_timer_hour">Message deletion timer is set to 1 hour.</string>
+    <string name="systemmsg_ephemeral_timer_day">Message deletion timer is set to 1 day.</string>
+    <string name="systemmsg_ephemeral_timer_week">Message deletion timer is set to 1 week.</string>
+    <string name="systemmsg_ephemeral_timer_four_weeks">Message deletion timer is set to 4 weeks.</string>
+    <string name="systemmsg_ephemeral_timer_disabled_by_me">Message deletion timer is disabled by me.</string>
```

Comment:
This one is not needed, there is a `systemmsg_action_by_user` which is already used.